CHANGES.rst

```diff
@@ -268,9 +212,42 @@ Contributor-facing changes
 
 
 
+Removals and backward incompatible breaking changes
+---------------------------------------------------
+
+- The shutdown logic in 3.9 waited on all tasks, which caused issues with some libraries.
+  In 3.10 we've changed this logic to only wait on request handlers. This means that it's
+  important for developers to correctly handle the lifecycle of background tasks using a
+  library such as ``aiojobs``. If an application is using ``handler_cancellation=True`` then
+  it is also a good idea to ensure that any :func:`asyncio.shield` calls are replaced with
+  :func:`aiojobs.aiohttp.shield`.
+
+  Please read the updated documentation on these points:
+  https://docs.aiohttp.org/en/stable/web_advanced.html#graceful-shutdown
+  https://docs.aiohttp.org/en/stable/web_advanced.html#web-handler-cancellation
+
+  -- by :user:`Dreamsorcerer`
+
+
+  *Related issues and pull requests on GitHub:*
+  :issue:`8495`.
+
+
+
+
 Miscellaneous internal changes
 ------------------------------
 
+- Improve performance of filtering cookies -- by :user:`bdraco`.
```

Comment:
@bdraco I'm usually trying to edit the change log before releasing to make its writing style consistent. Since it's a summary of news for people to find out what's happened since the previous release, the writing style should be different from Git commit messages. Imperative mood (“Improve”) reads as if it's a TODO-list time, not a statement on something that has happened. I sometimes recommend using the past tense or non-imperative present tense, referencing the differences between the versions.

This entry says “Improve”, the following one is “Improved”, followed by “Add”. They should all be consistent between each other, really. Hence, the need to rephrase entries.

Personally, I like doing this before generating the change log with Towncrier, just by checking the draft in the docs. This way, I can edit separate change notes and let Towncrier compose the end result w/o editing this file directly.

With pre-releases, it'd be possible to just revert the previous commit that did the generation, apply changes to all the fragments and generate a new commit. In the past, though, I'd branch out the pre-release change log commit without merging that into stable branches: this way, I could cut another release from a stable branch later on, without concerning myself with reverting the commit and getting the fragments back.

Reply:
I made 8535 a symlink to 7583

Reply:
Adjusted tense on all the ones that needed to to be past tense.